calculator.html

```diff
@@ -5,54 +5,94 @@
-            <button class="btn btn-block" data-bind="click: addNumber.bind(null, '0')">0</button>
-          </div>
-          <div class="col-xs-3">
-            <button class="btn btn-block" data-bind="click: addNumber.bind(null, '8')">8</button>
-            <button class="btn btn-block" data-bind="click: addNumber.bind(null, '5')">5</button>
-            <button class="btn btn-block" data-bind="click: addNumber.bind(null, '2')">2</button>
-            <button class="btn btn-block" data-bind="click: addMultiplication">&times;</button>
+        <div class="row">
+          <div class="col-xs-12">
+            <button class="btn btn-block btn-primary" data-bind="click: addCalculator">Add Calculator</button>
           </div>
-          <div class="col-xs-3">
-            <button class="btn btn-block" data-bind="click: addNumber.bind(null, '9')">9</button>
-            <button class="btn btn-block" data-bind="click: addNumber.bind(null, '6')">6</button>
-            <button class="btn btn-block" data-bind="click: addNumber.bind(null, '3')">3</button>
-            <button class="btn btn-block" data-bind="click: addDivision">&divide;</button>
+        </div>
+      </div>
+
+      <div class="col-lg-6 col-md-12">
+        <!-- ko with: toDoList -->
+        <div class="panel panel-primary">
+          <div class="panel-heading">
+            <h1>To-Do</h1>
           </div>
-          <div class="col-xs-3">
-            <button class="btn btn-block" data-bind="click: clear">C</button>
-            <button class="btn btn-block" data-bind="click: addPlus">+</button>
-            <button class="btn btn-block" data-bind="click: addMinus">-</button>
-            <button class="btn btn-block" data-bind="click: equal">=</button>
+          <div class="panel-body">
+            <h4 data-bind="visible: !tasks().length">You have nothing to do, great!</h4>
+            <div>
+              <ul class="list-group" data-bind="foreach: tasks">
+                <li class="list-group-item"><label><input type="checkbox" data-bind="checked: isSelected">
+                  <span data-bind="text: name, style: { color: isDone() ? 'green' : 'black' }">  </span></label>
+                </li>
+              </ul>
+
+            </div>
+            <div>
+              <button class="btn btn-success add-margin-btn" data-bind="click: markAsDone, visible: isMarkAsDoneVisible()">Make as done</button>
+              <button class="btn btn-success add-margin-btn" data-bind="click: markNotDone, visible: isMarkNotDoneVisible()">Make as not done</button>
+              <button class="btn btn-danger" data-bind="click: deleteTasks, visible: isDeleteVisible()">Delete</button>
+            </div>
+            <div class="task-text">
+              <form class="form-inline">
+                <div class="form-group">
+                  <input type="text" placeholder="Enter something to do.." class="form-control" data-bind="value: newTaskText">

```

Comment:
Enter something to do... (ellipsis is typically 3 dots not 2 https://en.wikipedia.org/wiki/Ellipsis)